client/go/vespa/load_env_test.go

```diff
@@ -0,0 +1,58 @@
+// Copyright Yahoo. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+package vespa
+
+import (
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestLoadEnv(t *testing.T) {
+	// setup
+	tmp := os.TempDir() + "/load_env_test.tmp"
```

Comment:
Nit: You can use `t.TempDir()` which is automatically created and removed after test ends.